### DISCUSSION TITLE
-bucket-file argumnt not work

### DISCUSSION THREAD

**Discussion by sr-if** [sa7mon/S3Scanner#321] [General]
### Existing Issues

- [X] I've searched the [current issues](https://github.com/sa7mon/S3Scanner/issues)

### Command

s3scanner -bucket-file test-s3

### Observed behavior (what happened?)

┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ docker run ghcr.io/sa7mon/s3scanner -bucket-file test-s3 
level=error msg="open test-s3: no such file or directory"
                                                                                                                                                                                        
┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ docker run ghcr.io/sa7mon/s3scanner -bucket-file ./test-s3 
level=error msg="open ./test-s3: no such file or directory"
                                                                                                                                                                                        
┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ docker run ghcr.io/sa7mon/s3scanner -version               
v3.0.4
                                                                                                                                                                                        
┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ cat test-s3
bucket123
assets
image-uploads

└─$ docker run ghcr.io/sa7mon/s3scanner -bucket-file /home/tools/test-s3/test-s3
level=error msg="open /home/tools/test-s3/test-s3: no such file or directory"

### Expected behaviour

sqs qs 

### Debug output

┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ docker run ghcr.io/sa7mon/s3scanner -bucket-file test-s3 
level=error msg="open test-s3: no such file or directory"
                                                                                                                                                                                        
┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ docker run ghcr.io/sa7mon/s3scanner -bucket-file ./test-s3 
level=error msg="open ./test-s3: no such file or directory"
                                                                                                                                                                                        
┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ docker run ghcr.io/sa7mon/s3scanner -version               
v3.0.4
                                                                                                                                                                                        
┌──(usersq㉿hzyuoj)-[/home/tools]
└─$ cat test-s3
bucket123
assets
image-uploads

└─$ docker run ghcr.io/sa7mon/s3scanner -bucket-file /home/tools/test-s3/test-s3
level=error msg="open /home/tools/test-s3/test-s3: no such file or directory"

**Comment by sa7mon** [comment]
You are not mounting the file into the container - any process inside that container has no visibility of files outside it. Use `-v` to bind mount a volume:

```
$ docker run -v $PWD:/data ghcr.io/sa7mon/s3scanner -bucket-file /data/test-s3
```

https://docs.docker.com/reference/cli/docker/container/run/#volume
